other/install_2-1_mysql.sql

```diff
@@ -1179,9 +1179,9 @@ CREATE TABLE {$db_prefix}user_likes (
 CREATE TABLE {$db_prefix}mentions (
   content_id INT DEFAULT '0',
   content_type VARCHAR(10) DEFAULT '',
-  id_mentioned INT DEFAULT 0,
-  id_member INT(10) UNSIGNED NOT NULL DEFAULT 0,
-  `time` INT NOT NULL DEFAULT 0,
+  id_mentioned INT DEFAULT '0',
```

Comment:
Mainly to be consistent.  Single quotes are used throughout all of these .sql files.  For installer & upgrader.  

Even the pg ones.  (I wish I could test pg, unfortunately I can't...)

The only exceptions are these 3 lines, where none were used, and which generate syntax errors.